treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -89,4 +89,19 @@
 
       Details of the alert can be found in the [alert summary]({{ alertHref }}), including links to graphs and comparisons for each of the affected tests. Please follow our [guide to handling regression bugs](https://wiki.mozilla.org/TestEngineering/Performance/Handling_regression_bugs) and **let us know your plans within 3 business days, or the offending patch(es) will be backed out** in accordance with our [regression policy](https://www.mozilla.org/en-US/about/governance/policies/regressions/).
 
+      For more information on performance sheriffing please see our [FAQ](https://wiki.mozilla.org/TestEngineering/Performance/FAQ).
+- model: perf.PerformanceBugTemplate
+  pk: 13
```

Comment:
I think it should be 8 here for the `pk` instead of 13. Is there a reason why it's 13?

Reply:
You're right. I missed that one.

Reply:
Based on the same logic, on line 33 should be `pk: 3` instead of `pk: 4`